lib/minesweeper_2pl/cli.rb

```diff
@@ -1,4 +1,35 @@
 module Minesweeper_2pl
   class CLI
+
+    def print(msg)
+      puts msg
+    end
+
+    def ask_for_move
+      string = "\nPlayer 1, enter one digit for the row (from left) and one digit for the column (from top) to make your move, eg. 3,1: "
+      self.print(string)
+    end
+
+    def get_player_input(game)
```

Comment:
It looks like there are a few responsibilities hidden in this method: getting input, validation, converting input to a coordinate representation. Would this method be more readable if you extracted these responsibilities to their own named methods?